lms/djangoapps/email_marketing/signals.py

```diff
@@ -82,7 +82,13 @@ def add_email_marketing_cookies(sender, response=None, user=None,
             post_parms['cookies'] = {'anonymous_interest': sailthru_content}
 
     time_before_call = datetime.datetime.now()
-    sailthru_response = get_email_cookies_via_sailthru.delay(user.email, post_parms)
+
+    try:
+        sailthru_response = get_email_cookies_via_sailthru.delay(user.email, post_parms)
+
+    except Exception as ex:
```

Comment:
Please remove this try/catch block and move this generic Exception for sailthru_response.get() call. Possible exception would be raised from sailthru_response.get() call, as the caller would be waiting the response or get any other exception.

Reply:
@adeelkhan please review again.I have updated it